app/controllers/state_file/questions/return_status_controller.rb

```diff
@@ -117,6 +119,21 @@ def reject_description
 
       def card_postscript; end
 
+      def redirect_if_no_submission
+        if current_intake.efile_submissions.empty?
+          redirect_to StateFile::Questions::InitiateDataTransferController.to_path_helper(us_state: state_code)
+        end
+      end
```

Comment:
Without this, an end use can break their whole intake by simply visiting the /return-status URL before it is ready.